docs/content/v2.20/sample-data/pgexercises.md

```diff
@@ -49,7 +49,7 @@ If you are using a local installation of YugabyteDB, run the `ysqlsh` command fr
 $ ./bin/ysqlsh
 ```
 
-If you are connecting to YugabyteDB Aeon, open the [ysqlsh cloud shell](/preview/yugabyte-cloud/cloud-connect/connect-cloud-shell/), or [run the YSQL connection string](/preview/yugabyte-cloud/cloud-connect/connect-client-shell/#ysqlsh) for your cluster from the `yugabyte-client` bin directory.
+If you are connecting to YugabyteDB Aeon, open the [ysqlsh cloud shell](/preview/yugabyte-cloud/cloud-connect/connect-cloud-shell/), or [run the YSQL connection string](/preview/yugabyte-cloud/cloud-connect/connect-client-shell/) for your cluster.
```

Comment:
Was this line redundant?

Reply:
On reflection, it seemed more clear to link to the instructions only rather than add this extra detail here.